extensions-contrib/kubernetes-overlord-extensions/src/test/java/org/apache/druid/k8s/overlord/KubernetesTaskRunnerTest.java

```diff
@@ -92,7 +104,10 @@
   private TaskLogPusher taskLogPusher;
   private DruidNode node;
 
-  public KubernetesTaskRunnerTest()
+  private final boolean useMultipleBaseTaskDirPaths;
+
+
+  public KubernetesTaskRunnerTest(boolean useMultipleBaseTaskDirPaths)
```

Comment:
Why would the kubernetes task runner be using multiple base dirs?  Isn't the whole idea that there's a pod allocated for the task and then the pod goes away, so there's no question of using multiple disks unless all of the directories are passed to the task and the task actively uses multiple directories, no?

Reply:
Not a k8s expert, but I think there could multiple volumes being shared and attached to multiple pods. So this code helps in doing round robin of volumes amongst various tasks/pods.

Reply:
1. you have to attach multiple volumes in the overlord. Tasks inherit spec from its parent, which is now the overlord. 
2. you would have to pass the multiple directories to the task, which is not happening, so it wont help at all. 
3. also there is no competition for the resources in k8s mode as the task is the only one writing to that volume, unlike MM-mode.  I don't think this would help in the future either as @imply-cheddar stated, you are still passing a single path to the peon.